django/db/models/sql/compiler.py

```diff
@@ -1110,6 +1110,8 @@ def prepare_value(self, field, value):
                 )
             if value.contains_aggregate:
                 raise FieldError("Aggregate functions are not allowed in this query")
+            if value.contains_over_clause:
+                raise FieldError('Window expressions are not allowed in this query')
```

Comment:
Dot is missing. Please add a dot to all exception messages.

Reply:
I just copied the pattern from the line above. I'll adjust the newly introduced messages in this PR.